README.md

```diff
@@ -45,6 +46,7 @@ There are two types of configs - schema config and user config. App detects auto
         "user": {
            "email": "maria@example.com",
            "business_schemas" : ["accounting", "sales"],
+           "business_schemas_write" : ["development"],
```

Comment:
auvajs, to je ošklivý, co třeba (bc compatible)
```
"business_schemas" : [
    "accounting", 
    {"name": "sales", "permissions": "read"}, 
    {"name": "development", "permissions": "write"}
]
```

Reply:
Proč ti to přijde lepší a proč je to první horší? Já mám takový ten "pattern matching" jemnocit. A míchání stringů a objektů v jednom poli mi ubližuje.

Reply:
Míchání stringů a objektů je tady jen kvůli zpětný kompatibilitě a třeba v AWS CF templatech se tohle běžne používá. 

Ošklivý to je proto, že za čas přibude další typ přístupu a bude se dělat další property, místo toho, aby se změnila value.

Reply:
Dohodli jsme se, že `business_schemas` zůstane v aktuálním stavu a bude deprekovaný. Pro zápis typu `{"name": "sales", "permissions": "read/write/cokoliv"}` bude nová property (`schemas`?).